lib/EpdFont/scripts/build-sd-fonts.py

```
@@ -0,0 +1,318 @@
+    cmd = [
+        sys.executable, str(manifest_script),
+        "--input", str(output_base),
+        "--base-url", base_url,
+        "--output", str(manifest_path),
+    ]
+
+    if config_path.exists():
+        cmd.extend(["--descriptions-from", str(config_path)])
+
+    manifest_path.parent.mkdir(parents=True, exist_ok=True)
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)